package.json

```diff
@@ -8,9 +8,12 @@
   },
   "scripts": {
     "compile": "truffle compile",
-    "test": "standard && truffle test",
-    "lint": "standard --fix",
-    "publish": "truffle publish"
+    "lint:js": "eslint --fix test/**/*.js migrations/**/*.js",
+    "lint:sol": "solhint contracts/*.sol contracts/*/*.sol test/*.sol test/*/*.sol",
+    "lint": "npm run lint:js && npm run lint:sol",
+    "publish": "truffle publish",
+    "test": "truffle test",
+    "posttest": "npm run lint"
```

Comment:
Can you make this pretest? Sometimes the linter finds bugs in the tests themselves. Useful to see that before the tests run.

Reply:
hmmm, is that generally agreed up on as the best approach? 
Not all devs will have a good eslint flow, so personally I find it easier to hack away and then lint afterwards. 
I suppose the `--fix` flag will minimize issues for them.

Reply:
I do actually feel strongly about this, it personally helps me a lot! I have a test that's failing, I don't know why, I spend time on it... If only I'd run my linter first, it tells me that something is `undefined`!

Reply:
Plus it forces you to exercise your coding style the whole time your working, instead of only seeing it as a "chore" at the end.

Reply:
your feelings are stronger than mine.